languages/en/default.xlf

```diff
@@ -7,7 +7,7 @@
     <file original="languages/en/default.xlf" source-language="en" datatype="plaintext">
         <body>
             <trans-unit id="MSC.duplicateMainLanguage">
-                <source>This main language record already has an assigment.</source>
+                <source>This main language record is already assigned to %s.</source>
```

Comment:
I also struggle with the word record or  Datensatz in the german translation because its quite hard to understand, what dataset the error message is refering to, What about something like "The selected page is already assigned to %s in the main language."

Reply:
I agree, can be changed imho :)